packages/relay-sdk/src/util.ts

```diff
@@ -0,0 +1,25 @@
+export function assertStatus(code: number, error: string) {
```

Comment:
Can we update `assertStatus` so that rather than ignoring the `response` entirely in cases where its `code` doesn't match what we expected, we instead _inspect_ it and include the actual reason why the request fails if the response included it? It would be preferable to inspect the response and only provide a generic string back to the caller if there was nothing returned by the backend service to pass along to the user.  [There's an example of this pattern here](https://github.com/LIT-Protocol/js-sdk/blob/f67160c79c9ae20f4b6d99628744c2dbe6b25207/packages/wrapped-keys/src/lib/service-client/utils.ts#L111)